src/Ztm.Zcoin.Rpc/ZcoinRpcClient.cs

```diff
@@ -362,70 +388,83 @@ static ushort ToNative(PropertyType type)
 
         class TransactionInfomation
```

Comment:
`TransactionInfomation` is ambiguous. IMO rename to `ExodusTransactionInformation`.